code/__HELPERS/mobs.dm

```diff
@@ -60,6 +63,29 @@
 
 		return f_style
 
+/proc/check_robolimb_appropriate(datum/sprite_accessory/S, organ_status, robolimb_manufacturer)
```

Comment:
DMDoc or at least a comment that tells what this proc does? Eg. "If a limb should be robotic [...] returns true if yes false otherwise"